tests/phpunit/tests/plugins/test-wp-importer.php

```diff
@@ -89,11 +89,31 @@ protected function _import_wp( $filename, $users = array(), $fetch_files = true
 	}
 
 	public function test_simple_import() {
-		$this->_import_wp( PLL_TEST_DATA_DIR . 'test-import.xml' );
-
+		$import_filepath      = PLL_TEST_DATA_DIR . 'test-import.xml';
+		$import_filepath_test = PLL_TEST_DATA_DIR . 'test-modified-import.xml';
+		$import_content  = strtr(
+			file_get_contents( $import_filepath ),
+			array(
+				'{lang_id_1}' => '<![CDATA[172]]>',
+				'{lang_id_2}' => '<![CDATA[174]]>',
```

Comment:
Since you use hard coded IDs, why the need to have formatted data with `lag_id_*`?

Reply:
Actually this is needed for the other test `test_simple_import_with_existing_languages()` because the languages are created before launching the import, so we need to inject the IDs into the XML file.

In this test, I've just restored the original IDs of the import file.